docs/appendix/upgrade-notes/2.x-to-3.0.rst

```diff
@@ -794,6 +794,11 @@ Entity and collection labelling conventions have changed to comply with the new
 
 These conventions are used across the routing and navigation systems, so plugins are advised to follow them.
 
+Request value filtering
+-----------------------
+
+``set_input()`` and ``get_input()`` no longer trim values.
```

Comment:
Sort of breaking, but I look at it as a bugfix. This should have never been trimmed.